pyproject.toml

```diff
@@ -22,7 +22,10 @@ include = [
 "cockpit.data" = ["*.html"]
 
 [tool.mypy]
-mypy_path = 'src'
+mypy_path = 'src/cockpit'
+[[tool.mypy.overrides]]
+module = ["systemd_ctypes.*"]
+follow_imports = 'silent'
```

Comment:
I need to remind myself, that I have correctly generated PYI files then we can get rid of this. But I assume that will take some back n forth with lis about how to keep those definitions in sync.

Reply:
But.. why? mypy has the entire source code readily at hand. It complains about things like 

    src/systemd_ctypes/inotify.py:48: error: "Type[inotify.Event@37]" has no attribute "CLOSE"

but how would PYI files fix any of these? Or are these hand-edited?

Reply:
mypy does not succeed running over systemd_ctypes either, as it exports classes dynamically.